documentation/howto/index.adoc

```diff
@@ -15,6 +15,8 @@ link:configure_remote_server.html[Configure a Remote Server]
 
 link:eapwfk_configure_ide.html[Configure JBoss EAP and JBoss Web Framework Kit for the IDE]
 
+link:cdk_howto.html[Using the IDE's Container Development Kit Tooling]
+
```

Comment:
Could we had the `CDK` acronym at the end of the link title, so user can search for 'CDK' in the page ?